### ISSUE TITLE
Icon component doesn't work with preact-boilerplate

### ISSUE THREAD

**Issue by mikaturk** [prateekbh/preact-material-components#105]
Hey,

I've been trying out preact-material-components along with preact-boilerplate but things aren't going so smoothly, I get this error when trying to use certain components (Icon being one of the many, they all give the same error)

```
webpack: Compiling...
Hash: 246d66e2749be1b08c22
Version: webpack 2.4.1
Time: 225ms
chunk    {0} bundle.js, 0.b603481d36874a65094c.hot-update.js (main) 481 kB [entry]
   [53] ./src/components/material/index.js 2.98 kB {0} [built]
  [123] ./~/css-loader?{"sourceMap":true,"importLoaders":1}!./~/postcss-loader?{}!./~/less-loader/dist?{"sourceMap":true}!./~/preact-material-components/
Icon/style.css 406 bytes {0} [built] [failed] [1 error]
     + 123 hidden modules

ERROR in ./~/css-loader?{"sourceMap":true,"importLoaders":1}!./~/postcss-loader?{}!./~/less-loader/dist?{"sourceMap":true}!./~/preact-material-components
/Icon/style.css
Module build failed:

      .mdc-icon-toggle--primary::before {
        background-color: color(var(--mdc-theme-primary, #3f51b5) a(14%)); } }
                        ^
Error evaluating function `color`: keyword.toLowerCase is not a function
      in /Users/mika/code/git/fswfototest/preact-boilerplate/node_modules/preact-material-components/Icon/style.css (line 425, column 26)
 @ ./~/preact-material-components/Icon/style.css 4:14-156 13:2-17:4 14:20-162
 @ ./src/components/material/index.js
 @ ./src/components/app.js
 @ ./src/index.js
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server ./index.js
Child html-webpack-plugin for "index.html":
    chunk    {0} index.html 543 kB [entry]
         + 4 hidden modules
webpack: Failed to compile.
```

**Comment by prateekbh** [comment]
Hey, can you post me your webpack's css config?

**Comment by mikaturk** [comment]
https://github.com/developit/preact-boilerplate/blob/master/webpack.config.babel.js
It's the bone stock one from preact-boilerplate

**Comment by prateekbh** [comment]
Cool will take a look

**Comment by mikaturk** [comment]
Hey, have you found the problem?

**Comment by prateekbh** [comment]
@mikaturk, will post it tomorrow morning.
Was off to some errands today

**Comment by mikaturk** [comment]
Ah nice, thanks 😄

**Comment by prateekbh** [comment]
hey!
I am building `preact-material-boilerplate` from the same repo. And fixed your issue for it.
For the same take a look at this commit
https://github.com/prateekbh/preact-material-boilerplate/commit/cc3b950d804f35e2bdfef1b5083b7163f2b74e17?diff=split

or if you wanna look at the final thing here's the delta

https://github.com/prateekbh/preact-material-boilerplate/commit/cc3b950d804f35e2bdfef1b5083b7163f2b74e17?diff=split


Basically, CSS Variables and less didn't played along very well

**Comment by mikaturk** [comment]
Ah ok, thanks 😄 
After copying the new file into my project, it gives this error: 
```
/mnt/d/Documenten/Code/fswfoto/webpack.config.babel.js:132
  plugins: [new _webpack2.default.NoEmitOnErrorsPlugin(), new _extractTextWebpackPlugin2.default({
            ^

TypeError: _webpack2.default.NoEmitOnErrorsPlugin is not a constructor
```

**Comment by prateekbh** [comment]
umm... although i didn't change any of that...

can you check what is your webpack's version in node_modules?

**Comment by mikaturk** [comment]
Aha, my dependencies were horribly outdated, copied the new ones from your package.json and it's all good, thanks so much for all the help man 👍

**Comment by prateekbh** [comment]
@mikaturk  no probs 👍.
Closing this issue, feel free to re-open if you feel like

**Comment by mikaturk** [comment]
update: cloned your new repository since I didn't change much and want maximum compatibility 😄

**Comment by prateekbh** [comment]
fyi... i just started it, will make it better soon.
